robotoff/insights/question.py

```diff
@@ -164,6 +168,62 @@ def get_source_image_url(barcode: str) -> Optional[str]:
 
         return None
 
+        def generate_selected_images(images: JSONType) -> JSONType:
+            selected_images = {}
```

Comment:
You should refactor this method to avoid code duplication (by looping over front/smal/thumb/display and languages)